stestr/repository/abstract.py

```diff
@@ -81,7 +81,7 @@ def get_inserter(self, partial=False):
             that testtools 0.9.2 and above offer. The startTestRun and
             stopTestRun methods in particular must be called.
         """
-        return self._get_inserter(partial)
+        return self._get_inserter(partial, run_id)
 
     def _get_inserter(self):
```

Comment:
So this is a weird case it's a method that doesn't really have a definition. I would have partial too, since that's missing as well. I can do that but I figured just leave it as is. But now that I think about it's kinda a weird to have an abstract class that doesn't define the actual interface for the method. I'll update this.